rasa/server.py

```diff
@@ -362,6 +371,8 @@ def configure_cors(
 
 
 def add_root_route(app: Sanic):
+    """Add / route to return hello."""
+
```

Comment:
```suggestion
```

Reply:
this was causing black to fail so reverted.